### DISCUSSION TITLE
I think sqlc can work with genji

### DISCUSSION THREAD

**Discussion by joe-getcouragenow** [sqlc-dev/sqlc#714] [Feature Requests & Ideas] [answered]
sqlc looks simple enough that i am sure it can work with genji.

Will require a little adjustment but not much because when writing SQL code with genji it feels very much like when using a normal SQL DB.

https://github.com/genjidb/genji

**Comment by kyleconroy** [comment, marked as answer]
Genji looks pretty cool. Since it's written in Go and the parser package is public (https://pkg.go.dev/github.com/genjidb/genji@v0.8.0/sql/parser), it would be possible to write a sqlc backend for it. I'm currently focusing on MySQL, PostgreSQL, and SQLite support and have no current plans for additional backends. If you;d like to work on it, I'm happy to help guide you.
